io.home-assistant.presence/dataset.tt

```diff
@@ -0,0 +1,62 @@
+dataset @io.home-assistant.presence language "en" {
+
+  program := now => @io.home-assistant.presence.state() => notify
+  #_[utterances=["is the presence sensor at home?",
+                 "check if the presence sensor is at home",
+                 "is the presence sensor away?",
+                 "check if the presence sensor is away",
+                 "check if the presence sensor is at home or away"]];
```

Comment:
"is the presence sensor at home?": where else would it be?

Reply:
For the presence sensor (https://developers.home-assistant.io/docs/en/entity_binary_sensor.html) the on state corresponds to home and off state corresponds to away. But yes the phrasing here is pretty bad, I'm not sure how to write an example.

Reply:
It would be something like "is anybody home" / "when nobody is home" / "when everybody leaves" etc.
Btw, this somewhat conflicts with "get_gps" (which has "when I am at ...", including "when I am at home"), as well as "security camera".

Reply:
okay should we remove this first in that case?

Reply:
removed for now